merlin/systems/triton/conversions.py

```diff
@@ -111,12 +117,19 @@ def _to_values_offsets(array):
     values, offsets
         Tuple of values and offsets
     """
-    num_rows = array.shape[0]
-    row_lengths = [array.shape[1]] * num_rows
-    offsets = [0] + list(itertools.accumulate(row_lengths))
     array_lib = cp if cp and isinstance(array, cp.ndarray) else np
+    shape = array.shape
+
+    values_shape = [-1]
+    if len(shape) > 2:
+        values_shape.extend(shape[2:])
+    values = array.reshape(*values_shape)
+
+    num_rows = shape[0]
+    row_lengths = [shape[1]] * num_rows if len(shape) > 1 else [1]
```

Comment:
What's the scenario where len(shape) is not going to be greater than 1 when calling this function? Does this [-1] need to be multiplied by num_rows also?

suggests we're converting from 1d array to a ragged list? Another option could be to perform an expand dims in the match representations function if we want to do this, which might be slightly clearer to understand the intent of the function that way?

Reply:
On further testing, it looks like these changes aren't actually required to get the example test passing, so I'm removing them from the PR.